### ISSUE TITLE
The reproduction is inconsistent with Kitti's official website

### ISSUE THREAD

**Issue by JunjieWang0528** [open-mmlab/OpenPCDet#1419]
When reproducing pv-rcnn, I used the official configuration file , a GPU and batch_size=2 to obtain the same test set as the following results.
![image](https://github.com/open-mmlab/OpenPCDet/assets/51773333/16be2357-e6e5-4db5-816a-37b214b31d16)
May I ask how to get another higher result on the leaderboard, as shown in the following figure.
![image](https://github.com/open-mmlab/OpenPCDet/assets/51773333/cff84380-6aaf-4708-a00e-7bc4344cd133)

**Comment by tyutlk** [comment]
I think when we talking benchmark,we are talking about validation set,not test set.

**Comment by JunjieWang0528** [comment]
> I think when we talking benchmark reproduced by openpcdet,we are talking about validation set,not test set.

Because I made improvements to PV-RCNN, there was an improvement in the validation set. The results obtained on the test set were better than the first screenshot, but not as good as the second screenshot. Therefore, I want to know where the problem may be.

**Comment by JunjieWang0528** [comment]
> I think when we talking benchmark reproduced by openpcdet,we are talking about validation set,not test set.

Thank you for your reply. I know where the problem lies

**Comment by bqcs** [comment]
> Thank you for your reply. I know where the problem lies

Can you please let me know what the issue is?

**Comment by Petros626** [comment]
@bqcs Kitti uses training and test for the benchmark, OpenPCDet uses training and validation I guess.
